### ISSUE TITLE
Custom backbone

### ISSUE THREAD

**Issue by Steffgroe** [kentaroy47/frcnn-from-scratch-with-keras#81]
Dear @kentaroy47 ,
I want to try to use your code with a smaller backbone for character recognition. 
I already copied the vgg.py file and try to define a simpel cnn using the same structure as the vgg file. I get the following errors:
Invalid argument: Incompatible shapes: [1,18,70,16] vs. [1,150,562,16]
	 [[{{node loss/rpn_out_regress_loss/rpn_loss_regr_fixed_num/sub}}]]
	 [[loss/Identity_1/_139]]
  (1) Invalid argument: Incompatible shapes: [1,18,70,16] vs. [1,150,562,16]
	 [[{{node loss/rpn_out_regress_loss/rpn_loss_regr_fixed_num/sub}}]]
0 successful operations.
0 derived errors ignored.
Where do I need to adjust shapes so the rpn losses are calculated correctly again?

**Comment by kentaroy47** [comment]
You may need to use a pretrained backbone to get sufficient accuracy.
Can you try mobilenet since I confirmed it working? Thanks!
